rets/http/client.py

```diff
@@ -59,6 +62,9 @@ def __init__(self,
         # this session id is part of the rets standard for use with a user agent password
         self._rets_session_id = ''
 
+        # initialize RETS-UA-Authorization with empty RETS Session ID (for GET transaction)
+        self._session.headers['RETS-UA-Authorization'] = 'Digest %s' % (self._user_agent_auth_digest())
```

Comment:
you could do `self._session.headers['RETS-UA-Authorization'] = _rets_ua_authorization()`

Reply:
Oops, did a rebase to get the original author's attribution. This line should be deleted now since I do it in the method (we can add caching into `_rets_ua_authorization()` if needed).